docs/DeveloperGuide.adoc

```diff
@@ -427,6 +427,102 @@ image::UndoRedoActivityDiagram.png[width="650" align="center"]
 ** Cons: This requires dealing with commands that have already been undone, in which we must remember to skip these commands. This violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
+//tag::Person Panel[]
+=== Person Panel
+The `PersonPanel` replaces the previous `BrowserPanel`, which is called upon using the methods in `CenterPanel`.
+`PersonPanel` will be used to display contact's information, groups and preferences depending on user's actions.
+
+==== Layout Implementation
+The specifications for the layout of PersonPanel is in `PersonPanel.fxml`. The figure below will show the visual layout of the `PersonPanel`.
```

Comment:
Can shorten “will show” to “shows”

Reply:
You can reference the figure number in a later PR when all pictures used in DG are finalized.